code/modules/admin/verbs/request_music.dm

```diff
@@ -0,0 +1,33 @@
+/mob/verb/request_music()
+	set category = "OOC"
+	set name = "Request Music"
```

Comment:
this needs a config.

rename it request internet sound. music has nothing to do with it.